src/panel/poi/PoiPanel.jsx

```diff
@@ -61,6 +61,7 @@ export default class PoiPanel extends React.Component {
   }
 
   componentDidMount() {
+    fire('mobile_direction_button_visibility', false);
```

Comment:
I expected we'd restore it in `componentWillUnmount`. Is it unnecessary? If so, it looks like a side-effect.

Reply:
it should not be necessary because the panel is resized when we quit a POI panel, hence the button animation is triggered.
However you are right it should probably be safer to trigger button visibility in componentDIdMount in case this behavior change in the futur.

Reply:
I update the code, thanks